news/_posts/2020-05-07-productboard-portal-launch.md

```diff
@@ -0,0 +1,200 @@
+Once again, you will have the possibility to comment and share your opinion on what is being built. You can even reach us out if you want to take part in any prototype testing and/or development. We will appreciate it a lot!
+
+
+
+Overall, Productboard will improve PrestaShop’s product development transparency and facilitate contributions from the community at every step of the process. From sharing an insight, to rating a need to solve, to contributing the user research, your voice will be heard and taken directly into account. That is the goal behind this move to Productboard.
+
+
+Here is the second part, where you’ll learn more about the PrestaShop Feedback interface.
+
+
+## How to use PrestaShop Feedback
+
+### See all features and expressions of needs, follow their development
+
+Each feature or expression of needs is represented by a card that includes a title and a description, sometimes an image.
+Click on a card to learn more about it.
+
+All cards fall into several tabs, each corresponding to a specific step in the development of a feature:```
+- Under consideration
+- In progress
+- Ready for the next release
+- Launched
+
+![PrestaShop Feedback home](/assets/images/2020/05/productboard-home.png)
+
+#### Under consideration
+Lists all cards that are already specified. At this point, rating it and let the team know why this feature is important helps us prioritize it.
+
+#### In progress
+Lists all features in progress and distributes them among five sections: User Research, Prototyping, Specifications, Development, QA testing.
+
+#### Ready for the next version
+Lists all features that are already developed and thus available on GitHub's latest branch. In short, it means those features will be accessible in the next minor release: 1.7.7, 1.7.8, etc.
+
+#### Launched
+Lists all released features.
+
+
+### Comment and vote for existing ideas
+When clicking on a card you can see 2 fields at its bottom. One to comment, and one to vote on the feature's impact **on you**.
+
+Voting for ideas is really important to help the team.
+Here is what happens behind the scenes:
+
+![PrestaShop Productboard user score](/assets/images/2020/05/productboard-user-score.png)
+
+When rating the impact of a feature, Productboard adds your vote to a “user score” that the team takes directly into account when evaluating features through its prioritization score.
+The more an idea is voted, the better rank it gets, and the more probability it has to integrate the software.
+
+This “user score” is not the only indicator that the team takes into account when evaluating a feature but it is key. Complexity, innovation, business impact for the merchants, legal aspects, are some of the criteria studied as well.
+
+Of course, the goal is not to vote “Critical” on every feature idea. This would cancel the benefits of the open votes.
```

Comment:
```suggestion
<div class="alert alert-note" role="alert">
Of course, the goal is not to vote “Critical” on every feature idea. This would cancel the benefits of the open votes.</div>
```